app/controllers/users_controller.rb

```diff
@@ -1,7 +1,9 @@
 class UsersController < ApplicationController
-  def index; end
+  def index
+    @users = User.all
```

Comment:
* Kindly fix this view as given in the wireframe, after creating users through the console i was able to view them 
![image](https://user-images.githubusercontent.com/52497246/178364024-3eb7894c-a002-48e5-948a-57e0f8b2e043.png)
![image](https://user-images.githubusercontent.com/52497246/178364047-b19b98c8-4c0f-41eb-b4aa-5f7242b28743.png)

Reply:
Dear @Davidosky007, I think there is a problem on your end. All 4 views are visible in my end:
<img width="1440" alt="Captura de Pantalla 2022-07-11 a la(s) 4 54 37 p  m" src="https://user-images.githubusercontent.com/75434179/178365493-cbd82d19-0eef-4413-a918-4fe705b6a031.png">
<img width="1440" alt="Captura de Pantalla 2022-07-11 a la(s) 4 54 39 p  m" src="https://user-images.githubusercontent.com/75434179/178365496-0aab7703-72d5-41f6-9afd-5156e376e1ab.png">
<img width="1440" alt="Captura de Pantalla 2022-07-11 a la(s) 4 54 41 p  m" src="https://user-images.githubusercontent.com/75434179/178365500-d42c96dc-083c-4a1d-ba40-1b73a9144e8e.png">
<img width="1440" alt="Captura de Pantalla 2022-07-11 a la(s) 4 54 45 p  m" src="https://user-images.githubusercontent.com/75434179/178365506-03f63fb5-3ec0-4ddf-a1b5-8f2b0e244bd3.png">